packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -302,6 +304,7 @@ export const messages = {
   error_overwrite_prompt: 'Error checking workspace for existing components',
   error_no_scratch_def:
     'No scratch definition files found. These files must be in the "config" folder and have file names that end with "-scratch-def.json". See [Scratch Org Definition File](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_def_file.htm) for help.',
```

Comment:
No scratch org definition found. The file must reside in the "config" folder and end with "-scratch-def.json". See...

@jag-sfdc - any reason to use plural? What happens if you have mult scratch defs that end in -scratch-def.json?Does VS Code prompt you to select one?

Reply:
Yes VS Code prompts to select one. I can change this.